### PR TITLE
Preparations for "release" :P

### PR DESCRIPTION
**Pull request by LucasMetal**
- README.md almost finished!
- Improved notification messages.
- Added some comments.

@mc15 take a look ;)

### CODE REVIEW

File: WashingBot.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ function setupUI(){
 
     txtConnection = app.CreateText("", 0.8, 0.2, "Multiline");
     lay.AddChild(txtConnection);
-    txtConnection.SetText("Waiting...");
+    txtConnection.SetText("Slack notifications disabled");
```
Comment (LucasMetal) [inline WashingBot.js]: Maybe this should read "Slack integration disabled". Because you can also send commands by chat, not only receive notifications, right?

File: README.md [hunk 2]
```diff
@@ -1 +1,129 @@
-# WashingBot
```
Comment (LucasMetal) [inline README.md:1]: Review this file in a new window (with nice rendering) ;)
I added a lot of stuff since you checked it.

File: README.md [hunk 3]
```diff
@@ -1 +1,129 @@
...
+- **Washing end is never detected:** make sure you have waited long enough for the threshold to be reached (`washingThresholdMinutes` in config.js), default is 10 minutes. You should also make sure that your phone's accelerometer doesn't report garbage when the screen goes off, you can use the UDP debugging for this (or the DroidScript IDE directly also).
+- **The cycle ends at lightning speed:** make sure `useSecondsInsteadOfMinutes` is disabled in the config :P
+- **Never connects to Slack:** make sure it's properly configured, see "Slack Integration" section.
+
+### Bugs, contact & contributions :shipit:
+
+- **Got a crazy idea/problem, fixed/found a bug or included a nice comment?** Just file an issue on GitHub or send a PR!!
+- **Have free time?** You can pick some ideas from the "Future improvements & ideas" section ;)  
+Idea: File an issue with the topic you want to work on before starting, just to coordinate (is this the right way to do it?).
+- **Just wanna contact us?** lucas.devescovi :recycle: gmail.com
```
Comment (LucasMetal) [inline README.md:120]: I didn't include your email, you can complete the Slack stuff, your email, and send another PR ;)

File: WashingBot.js [hunk 4]
```diff
@@ -166,13 +168,13 @@ function setupMonitor(){
...
-        txtStatus.SetText("Laundry finished!!! Duration: " + washingDurationMinutes + "\nWaiting for mulo...");
-        miniSlackBotInstance.sendMessage("Mulo a colgar la ropa!! :clock3: Duración del lavado: " + washingDurationMinutes);
+        txtStatus.SetText("Laundry finished!!! Duration: " + washingDurationMinutes + "\nWaiting for human...");
+        notifySlack("Human go hang the clothes!! :clock3: Washing duration: " + washingDurationMinutes);
         say("Laundry finished.");
     }
 
     config.onFinishedReminder = function(minsSinceFinish, reminderCount) {
-        miniSlackBotInstance.sendMessage(":information_source: " + reminderCount + " - Recordá colgar la ropa, hace " + minsSinceFinish + "mins. que terminó!");
+        notifySlack(":information_source: " + reminderCount + " - Remember to hang the clothes, washing has finished " + minsSinceFinish + "mins. ago!");
```
Comment (mc15) [inline WashingBot.js]: Aren't we missing a space here? (before "mins. ago!")

File: WashingBot.js (hunk 1 above)
Comment (mc15) [inline WashingBot.js]: I agree that it should say "integration" instead of "notifications", but I'm not quite sure I understand why you put this message here.
At this point you still don't know if the notifications have been disabled or not, maybe this message should be inside the setupSlackBot method and shown only if the slackToken was not provided.

Comment (LucasMetal) [inline WashingBot.js]: It's there because if integration is enabled, it will instantly change to "Waiting", the user will never see it.
But if it's disabled, it will remain as it was defined, for ever. It was a simple way of solving it. Of course it could be inside setupSlackBot function.

Do you think it's important to move it?

File: WashingBot.js (hunk 4 above)
Comment (LucasMetal) [inline WashingBot.js]: I really don't know, it was like that before. Do you think it would be better with the space?
":information_source: 2 - Remember to hang the clothes, washing has finished 5mins. ago!"
":information_source: 2 - Remember to hang the clothes, washing has finished 5 mins. ago!"

It's pretty the same for me, although it seems better with the space IMO.

File: README.md [hunk 5]
```diff
@@ -1 +1,129 @@
...
+----------------------
+
+A [DroidScript](https://play.google.com/store/apps/details?id=com.smartphoneremote.androidscriptfree) app to help you with your laundry. WashingBot monitors your washing machine, keeps you updated, reminds you to hang your clothes, talks and you can even chat with him!
+
+Low water pressure, different washing options and your mental state can affect the duration of your washing cycle, if you like us are too lazy to care and hate going to check in vane, then this app is for you!
+
+![WashingBot!](https://github.com/MetalBunker/WashingBot/blob/master/Img/WashingBot.png)
+  
+WashingBot feels the vibrations of your washing machine by using your cell's accelerometer, so he knows (or tries to guess :P) how is your washing going. He will notify you when your washing starts, finishes and he will even remind you to go and hang your clothes!  
+Notifications are verbal (yes, he likes to talk) and by chat, using [Slack](https://slack.com/). You can even ask him for how long he has been washing (yeah, he thinks he's actually doing the laundry, be nice and don't tell him).
```
Comment (mc15) [inline README.md:11]: "guess :P" => guess :stuck-out-tongue:

File: WashingBot.js (hunk 1 above)
Comment (mc15) [inline WashingBot.js]: Not really, we can leave it as it is for now.

File: WashingBot.js (hunk 4 above)
Comment (mc15) [inline WashingBot.js]: I like it more with the space :smile:

File: README.md (hunk 5 above)
Comment (mc15) [inline README.md:11]: After finishing reading, I realized that you haven't used emoticons everywhere. No need to change this, but if you do change it, please look for all the other places.

**Comment by mc15** [comment]
:+1:
